numpy/core/tests/test_getlimits.py

```diff
@@ -85,8 +90,29 @@ def test_finfo_repr(self):
 
 
 def test_instances():
-    iinfo(10)
-    finfo(3.0)
+    # Test the finfo and iinfo results on instances agree with the results
+    # on the corresponding classes 
+
+    for c in [int, np.int16, np.int32, np.int64]:
+        class_info = iinfo(c)
+        instance_info = iinfo(c(12))
+
+        for key in ['bits', 'min', 'max']:
+            assert getattr(class_info, key) == getattr(instance_info, key)
```

Comment:
Might make this a separate function  `assert_iinfo_equal`. I do love uniformity :)